Game/Source/GDKTestGyms/BenchmarkGymGameMode.h

```diff
@@ -35,6 +35,7 @@ class GDKTESTGYMS_API ABenchmarkGymGameMode : public AGameModeBase
 	double GetClientRTT() const { return AveragedClientRTTSeconds; }
 	double GetClientViewLateness() const { return AveragedClientViewLatenessSeconds; }
 	double GetPlayersConnected() const { return ActivePlayers; }
+	double GetFPSValid() const { return !bHasFpsFailed ? 1.0 : 0.0; }
```

Comment:
What sets bHasFpsFailed?

Reply:
This line here https://github.com/spatialos/UnrealGDKTestGyms/pull/100/files#diff-0fc3b032552e5b3294cf5ce6fb65fde4R289